schedule/staging/RAID1@64bit-staging-Y.yaml

```diff
@@ -0,0 +1,71 @@
+---
+name:           RAID1@64bit-staging
```

Comment:
s/staging/staging-Y/

Reply:
why change the name of the test, it's the same, no?